lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -2947,6 +2947,47 @@ mapping:
           This root domain will be written in the unique session cookie shared
           by all subdomains.
 
+      enable_tool_recommendations:
+        type: bool
+        default: false
+        required: false
+        desc: |
+          Allow the display of tool recommendations in workflow editor and after tool execution.
+          If it is enabled and set to true, please enable 'tool_recommendation_model_path' as well
+
+      tool_recommendation_model_path:
+        type: str
+        default: 'https://github.com/anuprulez/download_store/raw/tool_recommendation_model/tool_recommendation_model/tool_recommendation_model.hdf5'
+        required: false
+        desc: |
+          Set remote path of the trained model (HDF5 file) for tool recommendation.
+          It is required to be set if 'enable_tool_recommendations' property is enabled and set to true.
+          
+      topk_recommendations:
+        type: int
+        default: 20
+        required: false
+        desc: |
+          Set the number of predictions/recommendations to be made by the model
+
+      admin_tool_recommendations_path:
+        type: str
+        default: 'config/tool_recommendations_overwrite.yml'
```

Comment:
better:
```yaml
default: 'tool_recommendations_overwrite.yml'
path_resolves_to: config_dir
```
Otherwise it will be inside **a** config directory, not **the** config directory if `config` is overwritten.

All the cases in the schema that use `config/` in the path either mean a specific config relative to root (like `tour_config_dir`) or are handled as special cases here https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L751 .

Sorry, config can be rather confusing - I should add a short guide for devs soon.